packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -938,6 +961,31 @@ export class ReadAlongComponent {
         .subscribe((el_tag) => {
           // Only highlight when playing
           if (this.playing) {
+            //if auto pause is active and not on last word of the read along pause the audio
+            if (
+              this.autoPauseAtEndOfPage &&
+              el_tag in this.endOfPageTags &&
+              this.finalTaggedWord !== el_tag
+            ) {
+              //console.log(this.audio_howl_sprites.sound.seek() ,el_tag, this.endOfPageTags[el_tag], this.play_id);
+              //this is a work around to ignore id's not close to current sound time
+              //
+              const timeDiff = Math.round(
+                (this.audio_howl_sprites.sound.seek() -
+                  this.endOfPageTags[el_tag][0]) *
+                  100 //in microsecond
```

Comment:
I haven't tested yet, but this constant of 100 is surprising for what seems to be converting milliseconds to microseconds. Why not 1000?

Reply:
I will fix the comment to centi-seconds and not microseconds. I emailed you a verbose I used to guestimate the constant. As I was annotating it, 100 centiseconds seemed too large for an error or margin. I think it should be half.